### ISSUE TITLE
Augmentation

### ISSUE THREAD

**Issue by tlkvstepan** [JiaRenChang/PSMNet#8]
Hi, thank you for very impressive work!
Do you use augmentation (color, geometry) when training on KITTI? I see some augmentation code, but you did not mentioned that you use any augmentation in the paper.

**Comment by JiaRenChang** [comment]
@tlkvstepan 
We did not use data augmentation during training on KITTI.
We tried using data augmentation, and got worse results.

**Comment by tlkvstepan** [comment]
Thank you! I wonder is you network visibly over fit KITTI? Is test error much worse than training error?

**Comment by JiaRenChang** [comment]
@tlkvstepan 
Yes, there is overfitting.
We got  1.83% 3-px error in validation while 2.32% 3-px in testing.

**Comment by tlkvstepan** [comment]
Thanks!

**Comment by scholltan** [comment]
@JiaRenChang  hi, I wonder whether you have tried hard example mining or something like that when training on sceneflow dataset?

**Comment by JiaRenChang** [comment]
@scholltan 
Yes, we tried focal loss but it had no improvement.

**Comment by JiaRenChang** [comment]
@scholltan 
Yes, I agree your point, and I am trying multi-scale training to improve our results.
Multi-scale training often used in semantic segmentation, so I think it will be useful.
